readmeio/rdme
# fix: change docs command to always update all valid docs

## 🧰 Changes

* Fixes docs command issue where only some of the docs in a folder will update when an invalid doc is in the folder.
* Fixes issue where invalid docs displayed an error in terminal but did not specify which doc caused it.

On Main:
<img width="1144" alt="Screen Shot 2021-07-01 at 6 17 42 PM" src="https://user-images.githubusercontent.com/53894083/124224450-854b2a00-daba-11eb-9b29-560d1b0505ff.png">
This PR:
<img width="1281" alt="Screen Shot 2021-07-01 at 6 16 59 PM" src="https://user-images.githubusercontent.com/53894083/124224470-8da36500-daba-11eb-90d0-79ddb46028b6.png">


## 🧬 QA & Testing
For Docs Command:
- Check that all valid docs in a folder update in the project
- Check that the output to the console is easy to understand when an invalid doc is in folder
